include/analyzedphenotypes.admin.form.inc

```diff
@@ -4,7 +4,7 @@
  * Contains administrative interfaces of this module.
  */
 
-
+module_load_include('inc', 'analyzedphenotypes', 'api/analyzedphenotypes.projects.api');
```

Comment:
If it's included in the .module file then it doesn't need to be included anywhere else :-)